### PR TITLE
correction of gecodeerde lithologie

### PR DESCRIPTION
**Pull request by pjhaest**
small comments of Roel are solved. The larger subject of the flattened dataframe remains to be solved.

TODO: flatten results for multiple `hoofdgrondsoort`and `bijmenging
@Roel: what would be the best way to yield a flattened dataframe for these attributes (additional rows)?

### CODE REVIEW

**Comment by pjhaest** [comment]
@Roel : is this again a Travis issue? If easier, just delete the pull request and propose a new branch to ameliorate the current description of 'gecodeerde_lithologie', taking into account the multiple attributes. 
If this is in a flat dataframe with additional rows, or with additional tables is still open for discussion.

**Comment by Roel** [comment]
There seems to be an issue with the 'boolean' type of 'bijmenging_plaatselijk'. I didn't yet investigate why this fails.

**Comment by Roel** [comment]
The reason this fails is because the `bijmenging_plaatselijk` contains null values (NaN). Actually this is a faulty test: `bijmenging_plaatselijk` is defined as a _datatype: 'boolean'_ and _notnull: False_, so null values should be allowed.

The problem is that we check the Pandas `dtype` of the series: if the series contains NaN values this is generalised (upcast) from `boolean` to `object`, resulting in this test failure.

We should rework the test_search (maybe in the process splitting it up) to:
(a) test the _notnull_ of a field: if notnull is True the series must not contain NaNs
(b) test the _datatype_ of a field: taking (a) into account, the type of (remaining) values of the series should match the datatype of the field.

Suggestions to implement this are welcome, I'm not very familiar with Pandas series and dtype checking. (I tried `dropna()`, but this seems not to affect the dtype of the series).

**Comment by pjhaest** [comment]
Ok, this also seems an issue in the _datetime_ definition, e g.. the field `datum_aanvang` of the boreholes is defined as `object` in the resulting pandas dataframe. This requires a typecast for insertion in postgresql

**Comment by Roel** [comment]
> 
> 
> Regarding the inclusion of the second `hoofdgrondsoort` and the third `bijmenging` . Is it ok if I adjust the schema to include all columns by default (2 extra columns)? As mentioned, this will yield more empty cells in the dataframe, but the user can easily drop those if the data are not required.

See also #108

File: tests/data/update_test_data.py [hunk 1]
```diff
@@ -205,22 +205,22 @@ def update_file(filepath, url, process_fn=None):
     update_file('types/interpretaties/gecodeerde_lithologie'
                 '/gecodeerde_lithologie.xml',
                 'https://www.dov.vlaanderen.be/data/interpretatie/2001'
-                '-191079.xml')
+                '-199503.xml')
```
[the remark this answers is not in the document]
Comment (pjhaest) [reply tests/data/update_test_data.py]: :+1:

[the remark this answers is not in the document]
Comment (pjhaest) [reply tests/data/update_test_data.py]: :+1:

**Comment by pjhaest** [comment]
@Roel, @stijnvanhoey : any ideas on what went wrong with the Python 3.5 build?

**Comment by Roel** [comment]
Sometimes AppVeyor and Travis have errors running the tests (connection timeouts/resets etc.). I have restarted the errored Travis job.

**Comment by pjhaest** [comment]
Ok to merge?

**Pull request merged**